sql-cli/tests/test_project.py

```diff
@@ -0,0 +1,29 @@
+import tempfile
+
+from sql_cli.project import BASE_SOURCE_DIR, Project
+from tests.utils import list_dir
+
+BASE_PATHS_COUNT = 19
+BASE_PATHS = [obj for obj in list_dir(str(BASE_SOURCE_DIR)) if not obj.stem == ".gitkeep"]
+
+
+def test_initialise_project_with_dirname():
+    with tempfile.TemporaryDirectory() as dir_name:
+        assert not list_dir(dir_name)
+        Project(dir_name).initialise()
+        new_dir_files_list = list_dir(dir_name)
+        assert new_dir_files_list == BASE_PATHS
+        assert len(new_dir_files_list) == BASE_PATHS_COUNT
```

Comment:
```suggestion
        assert len(new_dir_files_list) == len(BASE_PATHS)
```
Why not do this^?

Reply:
Since we are getting the base directory from a configuration, I was worried that by mistake, we change it to an empty directory, and things would stop working. I thought having a counter for how many files/directories we want to copy would help avoid this kind of scenario.